v19.2/recommended-production-settings.md

```diff
@@ -23,42 +27,75 @@ Mentions of "CPU resources" refer to vCPUs, which are also known as hyperthreads
 
 ### Basic hardware recommendations
 
-Nodes should have sufficient CPU, RAM, network, and storage capacity to handle your workload. It's important to test and tune your hardware setup before deploying to production.
+This hardware guidance is meant to be platform agnostic and can apply to bare-metal, containerized, and orchestrated deployments. Also see our [cloud-specific](#cloud-specific-recommendations) recommendations.
+
+| Value | Recommendation | Reference
+|-------|----------------|----------
+| RAM per vCPU | 4 GB | [CPU and memory](#cpu-and-memory)
+| Capacity per vCPU | 60 GB | [Storage](#storage)
+| IOPS per vCPU | 500 | [Disk I/O](#disk-i-o)
+| MB/s per vCPU | 30 | [Disk I/O](#disk-i-o)
+
+Before deploying to production, test and tune your hardware setup for your application workload. For example, read-heavy and write-heavy workloads will place different emphases on [CPU](#cpu-and-memory), [RAM](#cpu-and-memory), [storage](#storage), [I/O](#disk-i-o), and [network](#networking) capacity.
 
 #### CPU and memory
 
-- At a bare minimum, each node should have **2 vCPUs and 2 GB of RAM**. More data, complex workloads, higher concurrency, and faster performance require additional resources.
+Each node should have **at least 2 vCPUs**, and for best performance we recommend between 4 and 32 vCPUs per node. Provision **4 GiB of RAM per vCPU**.
```

Comment:
nit/opinion: We use `GiB` here and `MB`, `GB` elsewhere. Is this intentional?

Reply:
Answered in below comment!